test/minimal_python_tests.yml

```diff
@@ -0,0 +1,22 @@
+version: "3.9"
+services:
+
```

Comment:
there is not action server running in this docker compose file, so how will the tests test the functionality of the action server?